### ISSUE TITLE
 Return an error message that indicates an infeasible value combination has been used. 

### ISSUE THREAD

**Issue by evanoffPNNL** [JGCRI/hectorui#24]
Return an error message that indicates an infeasible value combination has been used.

**Comment by kdorheim** [comment]
@crvernon and @bpbond I am slightly confused, this is a change to be made to the Hector UI or to the master branch of Hector?  Would it be possible to add a try-catch to the UI that if it returns 
```
Error: Error in hector::run(hcores[[i]], globalVars[["endDate"]]): Error while running hector: msg: Assertion failed: bad Tk value
func: ocean_csys_run
file: ocean_csys.cpp
ffile: ocean_csys.cpp
```

then it returns a more informative error message to the UI?

**Comment by bpbond** [comment]
Hmm. If `ocean_csys` crashes, how can we be sure it's due to bad parameter values?

**Comment by bpbond** [comment]
I mean, an unclear error message is better than a misleading one!

**Comment by crvernon** [comment]
Great point @bpbond .  Based on your point, it seems that we should simply constrain the bounds of the values in `hectorui` for now instead of adding the message until we can add some traceability to the current error.

**Comment by evanoffPNNL** [comment]
Fixed in 1.1.4
